platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/ChatterNetwork.java

```diff
@@ -0,0 +1,391 @@
+    private final ReconnectController reconnectController;
+    private final ReconnectThrottle reconnectThrottle;
+    private final Supplier<Consensus> consensusSupplier;
+    private final NotificationEngine notificationEngine;
+    private final StateManagementComponent stateManagementComponent;
+    private final FallenBehindManagerImpl fallenBehindManager;
+    private final boolean startedFromGenesis;
+    private final ReconnectMetrics reconnectMetrics;
+    private final ChatterEventMapper chatterEventMapper = new ChatterEventMapper();
+    private final QueueThread<EventIntakeTask> intakeQueue;
+    private final SequenceCycle<EventIntakeTask> intakeCycle;
+    private final EventCreatorThread eventCreatorThread;
+    private final ChatterEventCreator chatterEventCreator;
+    private final EventLinker eventLinker;
+
+    /**
+     * A list of threads that execute the chatter protocol.
+     */
+    private final List<StoppableThread> chatterThreads = new LinkedList<>();
+
+    /**
+     * Construct a chatter network.
+     */
+    public ChatterNetwork(
+            @NonNull final PlatformContext platformContext,
+            @NonNull final ThreadManager threadManager,
+            @NonNull final Crypto crypto,
+            @NonNull final Settings settings,
+            @NonNull final AddressBook initialAddressBook,
+            @NonNull final NodeId selfId,
+            @NonNull final SoftwareVersion appVersion,
+            @NonNull final NetworkTopology topology,
+            @NonNull final ConnectionTracker connectionTracker,
+            @NonNull final Time time,
+            @NonNull final NetworkMetrics networkMetrics,
+            @NonNull final SyncMetrics syncMetrics,
+            @NonNull final EventTaskCreator eventTaskCreator,
+            @NonNull final EmergencyRecoveryManager emergencyRecoveryManager,
+            @NonNull final ShadowGraph shadowGraph,
+            @NonNull final ReconnectController reconnectController,
+            @NonNull final ReconnectThrottle reconnectThrottle,
+            @NonNull final Supplier<Consensus> consensusSupplier,
+            @NonNull final NotificationEngine notificationEngine,
+            @NonNull final StateManagementComponent stateManagementComponent,
+            @NonNull final FallenBehindManagerImpl fallenBehindManager,
+            @NonNull final SwirldStateManager swirldStateManager,
+            final boolean startedFromGenesis,
+            @NonNull final ReconnectMetrics reconnectMetrics,
+            @NonNull final StartUpEventFrozenManager startUpEventFrozenManager,
+            @NonNull final FreezeManager freezeManager,
+            @NonNull final CriticalQuorum criticalQuorum,
+            @NonNull final QueueThread<EventIntakeTask> intakeQueue,
+            @NonNull final SequenceCycle<EventIntakeTask> intakeCycle,
+            @NonNull final EventObserverDispatcher dispatcher,
+            @NonNull final EventLinker eventLinker) {
```

Comment:
31 params 😳 Can we somehow change this? Maybe be wrapping params together in records?